files/default/nginx-status.conf

```diff
@@ -0,0 +1,12 @@
+server {
+  listen 127.0.0.1:81;
+  server_name localhost;
+
+  access_log off;
+  allow 127.0.0.1;
+  deny all;
+
+  location /nginx_status {
+    stub_status;
+  }
+}
```

Comment:
This creates an endpoint that outputs some simple nginx stats. I lifted it from some DataDog setup instructions, but it's not datadog specific. I'd like to have it enabled in case we want to collect these metrics at some point.

Output example:
```
Active connections: 43 
server accepts handled requests
 7368 7368 10993 
Reading: 0 Writing: 5 Waiting: 38
```